.github/workflows/shared_deploy_cloudhub_1_0.yml

```
@@ -62,7 +64,7 @@ jobs:
           maven_settings_path: ${{ inputs.maven_settings_path }}
 
       - name: Deploy to CloudHub
-        uses: nimblehq/mulesoft-actions/deploy_cloudhub_1_0@v1.6
+        uses: nimblehq/mulesoft-actions/deploy_cloudhub_1_0@v1.14
```

Let's remove all `not-working/testing` releases - rename the `1.14` to `1.17` @tednguyendev 

```suggestion
        uses: nimblehq/mulesoft-actions/deploy_cloudhub_1_0@v1.7
```

---

Resolved here ✋ 5cfdd72092706369c5dc629b5d69de2ce60a6ff4